src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2843,7 +2843,9 @@ private BoundExpression BindDelegateCreationExpression(ObjectCreationExpressionS
                 bool hasErrors = false;
                 if (analyzedArguments.HasErrors)
```

Comment:
Removing this introduce a lot of extra `ERR_MethodNameExpected` for the existing test cases.